Mathlib/Data/List/Sort.lean

```diff
@@ -377,39 +377,39 @@ theorem perm_split : ∀ {l l₁ l₂ : List α}, split l = (l₁, l₂) → l ~
     exact ((perm_split e).trans perm_append_comm).cons a
 
 /-- Implementation of a merge sort algorithm to sort a list. -/
-def mergeSort : List α → List α
+def mergeSort' : List α → List α
```

Comment:
Is `mergeSort` now upstream? Can you add a comment/todo about the status of this?

Reply:
I've written a few paragraphs in the module doc.